### PR TITLE
Bump dev tools to latest versions.

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordon.smith@lexisnexis.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@mlzummo Please Review

File: package.json [hunk 1]
```diff
@@ -3,30 +3,29 @@
...
-    "requirejs": "~2.1.17",
-    "del": "~1.1.1",
-    "csso": "~1.3.11",
-    "gulp-git": "~1.2.3",
-    "gulp-bump": "~0.3.0",
-    "gulp-filter": "~2.0.2",
-    "gulp-tag-version": "~1.2.1",
-    "yargs": "~3.8.0",
+    "gulp-util": "~3.0.7",
+    "lodash": "~3.10.1",
```
Comment (mlzummo) [inline package.json]: where are we using lodash?

**Comment by mlzummo** [comment]
looks good, no changes to shrink wrap need to be made?

**Comment by GordonSmith** [comment]
I deleted shrinkwrap as the latest dev files have no issues.

**Pull request merged**